pkg/runner/test_runner.go

```diff
@@ -548,14 +550,28 @@ func (tr *TestRunner) WaitPipelineCompletion(terminate <-chan struct{}, ch compl
 			return nil
 		case res := <-ch.routingResultCh:
 			err = res.err
-			if err != nil {
-				err = fmt.Errorf("error at test step '%s' (label: '%s'): %w", res.bundle.TestStep.Name(), res.bundle.TestStepLabel, err)
-			}
 			tr.state.SetRouting(res.bundle.TestStepLabel, res.err)
 		case res := <-ch.stepResultCh:
 			err = res.err
 			if err != nil {
-				err = fmt.Errorf("error at test step '%s' (label: '%s'): %w", res.bundle.TestStep.Name(), res.bundle.TestStepLabel, err)
+				rm := json.RawMessage("\"" + err.Error() + "\"")
```

Comment:
since this will be JSON-encoded, I need to add quoting to the string or encoding will fail. This can still fail if there is a literal quote in the error string. Is there a safe way to escape it?

Reply:
I think you can just put it into `json.Marshal(err.Error())`